docs/embed-rust.md

```diff
@@ -3,7 +3,7 @@
 This document shows an example of how to embed Wasmtime using the [Rust
 API][apidoc] to execute a simple wasm program. Be sure to also check out the
 [full API documentation][apidoc] for a full listing of what the [`wasmtime`
-crate][crate] has to offer.
+crate][wasmtime] has to offer.
```

Comment:
Would it make sense to mention the examples here too, just to make sure people who skip ahead to the embedding chapter don't miss the examples that go along with it?

Reply:
Sure!